content/en/docs/managing-jx/old/create-cluster.md

```diff
@@ -371,4 +371,4 @@ Now **[develop apps faster with Jenkins X](/docs/getting-started/next/)**.
 
 ## Troubleshooting
 
-If you hit any issues installing Jenkins X then please check out our [troubleshooting guide](/troubleshooting/faq/) or [let us know](/community) and we'll try our best to help.
+If you hit any issues installing Jenkins X then please check out our [troubleshooting guide](https://jenkins-x.io/troubleshooting/faq/) or [let us know](/community/) and we'll try our best to help.
```

Comment:
that page also doesn't exist in that incarnation anymore. might be best to link to /managing-jx/faq/